index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Sofia"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <nav class="logo">
+          <a href="#" class="logo1"> My Logo </a>
+          <div class="menu"><img src="Menu.png" alt="" /></div>
+        </nav>
+      </header>
+
+      <section class="content">
+        <div>
+          <h1 class="heading">I am Saif glad to see you!</h1>
+        </div>
+        <div>
+          <div class="texto">
+            <p class="para_text">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
```

Comment:
- [ ] OPTIONAL: I recommend you using the same font family as provided in figma, which is `Poppins` to match your design with original design